docs/control_plane/device_templates/CONFIGURE_NETWORK_SWITCHES.md

```diff
@@ -0,0 +1,32 @@
+# Configuring Dell EMC Networking Switches  
```

Comment:
Let's retitle this and rename the file. Instead of "Network Switches" it should be "PowerSwitch"

Reply:
It is renamed to PowerSwitch.